media/js/app/projects/selection_assignment_edit.js

```diff
@@ -158,11 +158,13 @@
             var pk = jQuery(this.hoverItem).attr('data-id');
```

Comment:
this.hoverItem is referenced above the check.

Theoretically, the user shouldn't be able to select something when there is no hoverItem. Unless there's a race condition between mouseOut & the selection.